test/output/3.sh.txt

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+ff()
+{
+  while read FILE; do
+    echo "$FILE"
+
+  done <<END
+	$(list "$@")
```

Comment:
I think this formatting is actually not valid. There is still a tab character instead of spaces.